dirsrvtests/tests/suites/import/import_test.py

```diff
@@ -246,7 +277,7 @@ def test_ldif2db_allows_entries_without_a_parent_to_be_imported(topo, _import_cl
     topo.standalone.start()
 
 
-def test_ldif2db_syntax_check(topo):
+def test_ldif2db_syntax_check(topo, _import_clean):
```

Comment:
Added _import_clean to remove syntax error ldif file used for ldif2dn offline test.